conanfile.py

```diff
@@ -1,60 +1,14 @@
 from conans import ConanFile, CMake
 import sys, os, subprocess
 
-# Conan evaluates the conanfile's "version" attribute twice: once when
-# doing "conan export" and again after the conanfile has been copied over
-# to the Conan package store. When it happens the second time, the conanfile
-# is no longer in a git repo, so the call to "git describe" to get the version
-# through the git tag doesn't work.
-#
-# This function detects whether or not the conanfile is being evaluated from
-# a repo (the first time) or the package store (the second time). If from a repo,
-# it gets the version via "git describe" and writes the version to a VERSION file.
-# If from the package store, it checks the VERSION file.
-#
-def get_version():
+sys.path.append(os.path.abspath(os.path.join(os.environ['CONAN_USER_HOME'], 'lib')))
+from polysync_conan_utils import *
```

Comment:
This won't work if external folks try to run conan, but it didn't work for them before.